config/controllers/posts_controller.rb

```diff
@@ -0,0 +1,5 @@
+class PotsController < ApplicationController
+  def index; end
+
+  def show; end
+end

```

Comment:
- [ ] `posts_controller.rb` file suppose to be inside `./app/controllers` directory, not inside `./config/controllers/` as you did. please fix that.